src/host.rs

```diff
@@ -54,7 +54,7 @@ impl Host {
         }
     }
 
-    pub(crate) fn send(
+    pub(crate) fn embark(
```

Comment:
I appreciate the name change, and this is better than `send`, but it still sucks.

Reply:
Name or structure?